run.py

```diff
@@ -165,10 +207,11 @@ def nonbib_to_master_pipeline(nonbib_engine, schema, batch_size=1):
     tmp = []
     i = 0
     max_rows = config['MAX_ROWS']
-    q = session.query(models.NonBibTable).options(load_only(*nonbib_to_master_fields))
+    q = session.query(models.NonBibTable).options(load_only(*nonbib_to_master_select_fields))
     for current_row in q.yield_per(100):
         current_row = nonbib_to_master_dict(current_row)
         add_data_link(session, current_row)
+        cleanup_for_master(current_row)
```

Comment:
if you are only asking for fields that are sent to master, why is it necessary to remove them? something to do with 'coalescence'?

Reply:
we use many of the boolean fields to set the 'property' value that goes to master, but they are not sent to master as individual fields.  so we request them, use them, and then delete them before we construct the protobuf.